docs/src/main/asciidoc/datastore.adoc

```diff
@@ -71,7 +71,10 @@ The following configuration options are available:
 | `spring.cloud.gcp.datastore.credentials.encoded-key` | Base64-encoded OAuth2 credentials for authenticating with the Google Cloud Datastore API, if different from the ones in the <<spring-cloud-gcp-core,Spring Cloud GCP Core Module>> | No |
 | `spring.cloud.gcp.datastore.credentials.scopes` | https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Cloud GCP Cloud Datastore credentials | No | https://www.googleapis.com/auth/datastore
 | `spring.cloud.gcp.datastore.namespace` | The Cloud Datastore namespace to use | No | the Default namespace of Cloud Datastore in your GCP project
-| `spring.cloud.gcp.datastore.emulator-host` | The `hostname:port` of the https://cloud.google.com/datastore/docs/tools/datastore-emulator[Datastore Emulator] to connect to | No |
+| `spring.cloud.gcp.datastore.host` | The `hostname:port` of the datastore service to connect. Can be used to connect to a https://cloud.google.com/datastore/docs/tools/datastore-emulator[Datastore Emulator]. If emulator are enabled and this property are empty, it will used as `localhost:<emulator_port>` | No |
```

Comment:
I don't think this setting should be here. We're not advocating using any Cloud Datastore host other than the real one the client library uses by default. The only other host I think is reasonable to specify is the emulator, but you already have 2 settings for that below.

Reply:
@meltsufin WDYT

Reply:
I agree with that. Do you guys think people will use the emulator on a different host than the client using the emulator itself?

Reply:
I agree with that. Do you think people will use the emulator on a different host? If not, we can just have the port and always connect to the localhost if the auto emulator is enabled.

If there is a chance to people trying to connect to external emulators, then we need to think abouut this property.